webpack.config.js

```diff
@@ -1,4 +1,5 @@
 var path = require('path');
+var glob = require("glob");
```

Comment:
allows you to do wildcard matching. when we add more test files, they will automatically get run. (see line 12)